_opensearch/query-dsl/index.md

```diff
@@ -120,3 +120,38 @@ With query DSL, however, you can include an HTTP request body to look for result
 }
 ```
 The OpenSearch query DSL comes in three varieties: term-level queries, full-text queries, and boolean queries. You can even perform more complicated searches by using different elements from each variety to find whatever data you need.
+
+## A note on special characters in field values 
+
+Due to word boundaries associated with Unicode special characters, the Unicode standard analyzer cannot index [text field type](https://opensearch.org/docs/2.2/opensearch/supported-field-types/text/) values as whole values when they contain these special characters. As a result, a field value of this type that includes a special character is parsed by the standard analyzer as multiple values separated by the special character, effectively tokenizing the different elements either side of it. This can lead to unintentional filtering of documents and potentially compromise the safeguards that control access. 
```

Comment:
I would rewrite this sentence for clarity as follows:

The Unicode standard analyzer can't index a String that contains a special character such as a Unicode character.

For example, if you create a string search using the `text` field type, and the string contains a special character, the standard analyzer parses each character as an individual value. This tokenizes each element on either side of the special character. 
When text fields are broken into individual values, it leads to unintentional document filtering and can compromise the control access safeguards.

To learn more about the String field types, see [text field type](https://opensearch.org/docs/2.2/opensearch/supported-field-types/text/)

Reply:
"...such as a Unicode character." Apparently the standard analyzer has problems with these specific special characters, so I don't want to say "such as". There's a reference to the exact special characters at the end of the note.
"For example, if you create a string search using the `text` field type..." I want the reader to know that the issue is with the `text` field type specifically. So would like to leave it in first sentence. Using `keyword` is mentioned later as an alternative to this.
I massaged some other parts of the note based on these comments. Thanks.